### PR TITLE
JAMES-2526 UnauthorizedEndpoint can be performed on a single James se…

### PR DESCRIPTION
**Pull request by chibenwa**
…rver

As all the endpoints are rejected with a 401 unauthorized we can clearly
play all 89 the tests on the same James instance, saving 88 restarts

### CODE REVIEW

File: server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/CassandraJmapExtension.java [hunk 1]
```diff
@@ -43,19 +46,71 @@
...
+        JamesLifeCyclePolicy EACH = serverSupplier -> new JamesLifecycleHandler(
+            Optional::empty,
+            () -> Optional.of(serverSupplier.get()),
+            GuiceJamesServer::stop,
+            guiceJamesServer -> { });
+        JamesLifeCyclePolicy ALL = serverSupplier -> new JamesLifecycleHandler(
+            () -> Optional.of(serverSupplier.get()),
+            Optional::empty,
+            guiceJamesServer -> { },
+            GuiceJamesServer::stop);
```
Comment (rouazana) [inline server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/CassandraJmapExtension.java]: the naming is a little hard to get out of context. s/EACH/FOR_EACH_TEST/ s/ALL/COMMON_TO_ALL_TEST/ ? Or something like that.

Comment (rouazana) [reply server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/CassandraJmapExtension.java]: What about having a builder for JamesLifecycleHandler, the parameters are really not straightforward.

**Comment by Arsnael** [comment]
```
[41f5f66cf6cd34f59e62d7a1e2d43d2821d2a1fb] [ERROR] Errors: 
[41f5f66cf6cd34f59e62d7a1e2d43d2821d2a1fb] [ERROR]   UnauthorizedEndpointsTest.setup:59 NullPointer
[41f5f66cf6cd34f59e62d7a1e2d43d2821d2a1fb] [ERROR]   UnauthorizedEndpointsTest.setup:59 NullPointer
[41f5f66cf6cd34f59e62d7a1e2d43d2821d2a1fb] [ERROR]   UnauthorizedEndpointsTest.setup:59 NullPointer
[...]
```

**Comment by Arsnael** [comment]
I'm not sure where to put that in your design, but I think you clearly miss a start of the James server somewhere, thus the `NullPointer`

**Comment by chibenwa** [comment]
I fixed it: see the use of the `static` keyword

**Comment by chibenwa** [comment]
Before

```
[INFO] Apache James :: Server :: Web Admin server integration tests 3.4.0-SNAPSHOT SUCCESS [05:35 min]
```

After

```
[INFO] Apache James :: Server :: Web Admin server integration tests 3.4.0-SNAPSHOT SUCCESS [02:53 min]
```

Gain 2 min 40

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**